articles/azure-functions/functions-reference-powershell.md

```diff
@@ -52,13 +52,13 @@ PSFunctionApp
  | - bin
 ```
 
-At the root of the project, there's a shared [host.json](functions-host-json.md) file that can be used to configure the function app. Each function has a folder with its own code file (.ps1) and binding configuration file (function.json). The name of `function.json`'s parent directory is always the name of your function.
```

Comment:
This gets rendered as:

`[host.json](functions-host-json.md)`

Instead of what I think you want which is

[`host.json`](functions-host-json.md)

Backticks inside of []

Reply:
Ah, missed that one. Got it updated.